core/src/main/resources/jenkins/model/Messages.properties

```diff
@@ -76,3 +76,5 @@ ItemCategory.Uncategorized.DisplayName=Uncategorized
 ItemCategory.Uncategorized.Description=Item types that have not yet been categorized by their plugin maintainer.
 ItemCategory.StandaloneProjects.DisplayName=Standalone Projects
 ItemCategory.StandaloneProjects.Description=Create projects with a self-contained configuration and history. These projects can be at the top-level or grouped within folders.
+ItemCategory.NestedProjects.DisplayName=Nested Projects
+ItemCategory.NestedProjects.Description=Create project categories or project hierarchies with folders. Folders can be created manually or automatically based on repositories.
```

Comment:
@daniel-beck This description was provided by @HRMPW. By now, the categories are not shown.